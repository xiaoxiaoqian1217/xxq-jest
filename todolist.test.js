import React from 'react';
import renderer from 'react-test-renderer'; // 快照测试
import TodoList from './todolist'
it('renders correctly', () => {
  const tree = renderer
  .create(<TodoList  />)
          .toJSON();

  expect(tree).toMatchSnapshot();
});