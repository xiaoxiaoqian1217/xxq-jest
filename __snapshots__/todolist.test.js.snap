// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="todo-list"
>
  <div>
    <span
      className="item-text "
    >
      1
    </span>
    <button
      onClick={[Function]}
    >
      done
    </button>
  </div>
  <div>
    <span
      className="item-text "
    >
      2
    </span>
    <button
      onClick={[Function]}
    >
      done
    </button>
  </div>
</div>
`;
